src/main/java/ch/njol/skript/conditions/CondIsSwimming.java

```diff
@@ -0,0 +1,56 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.conditions;
+
+import org.bukkit.entity.LivingEntity;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+
+/**
+ * @author Peter Güttinger
+ */
+@Name("Is Swimming")
+@Description({"Checks whether a living entity is swimming.",
+		"Note: Only supported on Minecraft 1.13+"})
+@Examples("player is swimming")
+@Since("INSERT VERSION")
+public class CondIsSwimming extends PropertyCondition<LivingEntity> {
+	
+	static {
+		if (Skript.methodExists(LivingEntity.class, "isSwimming"))
+			register(CondIsSwimming.class, "swimming", "entities");
```

Comment:
You may want to change it to `livingentities` instead of just entities since only living entities can swim.

Reply:
I swear I had that in there and I got an error, I think I changed the wrong thing!
Fixed